### ISSUE TITLE
Some vowel combinatons of Telugu TSA and DZA don't work properly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [googlefonts/noto-fonts#346]
```
It seems to me from some early testing that some of the combinations on 
http://www.unicode.org/L2/L2015/15082-telugu-sym.pdf don't work right. 
Assigning to myself to investigate.
```

Original issue reported on code.google.com by `roozbeh@google.com` on 24 Apr 2015 at 7:38

**Comment by behdad** [comment]
cc @roozbehp @waksmonskiMT

**Comment by marekjez86** [comment]
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">(పచచ్ǵ)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">(పచచ్ǵ)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">(సజజ్లు)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">(సజజ్లు)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">ȇౘ◌్చుకుǵ (ȇచుచ్కుǵ)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">ȇౘ◌్చుకుǵ (ȇచుచ్కుǵ)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">అౘ◌్చంĦా (అచచ్ంĦా)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">అౘ◌్చంĦా (అచచ్ంĦా)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">కూౙ◌ా (కూజా)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">కూౙ◌ా (కూజా)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">పౘ◌్చǵ</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">పౘ◌్చǵ</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">సౙ◌్జలు</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">సౙ◌్జలు</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">ౘ◌ాకȃ (Ĩాకȃ)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">ౘ◌ాకȃ (Ĩాకȃ)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">ౘ◌ుకక్లు (చుకక్లు)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">ౘ◌ుకక్లు (చుకక్లు)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">ౘ◌ూపులు (చూపులు)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">ౘ◌ూపులు (చూపులు)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">ౙ◌ాǹȃ (జాǹȃ)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">ౙ◌ాǹȃ (జాǹȃ)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">ౙ◌ు◌ంటి (ǯంటి)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">ౙ◌ు◌ంటి (ǯంటి)</span></p>
<p>NotoSansTelugu </p><p><span style="font-family:NotoSansTelugu, AdobeBlank; font-weight:400; background-color:#EDEBEA">ౙ◌ూలు (ǰలు)</span></p>
<p>NotoSerifTelugu </p><p><span style="font-family:NotoSerifTelugu, AdobeBlank;  font-weight:400; background-color:#EDEBEA">ౙ◌ూలు (ǰలు)</span></p>

**Comment by marekjez86** [comment]
@nizarsq : Nizar could you prepare a test case with the above text and assign it back to me? Thanks

**Comment by nizarsq** [comment]
> @nizarsq : Nizar could you prepare a test case with the above text and assign it back to me? Thanks

Sure. I'll update the bug when the test case ready. Thanks

**Comment by nizarsq** [comment]
Test case created.
<img width="644" alt="Screen Shot 2020-08-03 at 6 19 27 PM" src="https://user-images.githubusercontent.com/22158882/89242220-dfdfbb80-d5b5-11ea-8a80-cd4d0781aeab.png">

**Comment by JelleBosmaMT** [comment]
I am getting e-mail messages about this. A test case for this issue looks like this:
<img width="642" alt="image" src="https://user-images.githubusercontent.com/12729011/89265757-41933a80-d635-11ea-8bf2-5c14505b6c46.png">
First and third line compare with original without modifier on the second and fourth lines. This is the Serif. The Sans has the same glyph set and the same rendering rules. The picture is from Mac Pages.  The text is:
 ౘా ౘి ౘీ ౘు ౘూ ౘృ ౘౄ ౘె ౘే ౘై ౘొ ౘో ౘౌ ౘ్ ౘ్ద ౘ్ౘ ద్ౘ ౘ్ర
చా చి చీ చు చూ చృ చౄ చె చే చై చొ చో చౌ చ్ చ్ద
ౙా ౙి ౙీ ౙు ౙూ ౙృ ౙౄ ౙె ౙే ౙై ౙొ ౙో ౙౌ ౙ్  ౙ్ద ౙ్ౙ ద్ౙ ౙ్ర
జా జి జీ జు జూ జృ జౄ జె జే జై జొ జో జౌ జ్ జ్ద 
ౚా ౚి ౚీ ౚు ౚూ ౚృ ౚౄ ౚె ౚే ౚై ౚొ ౚో ౚౌ ౚ్ర ౚ్ద
ఴా ఴి ఴీ ఴు ఴూ ఴృ ఴౄ ఴె ఴే ఴై ఴొ ఴో ఴౌ ఴ్ ఴ్ద
This issue can be closed!

**Comment by moyogo** [comment]
Fixed in Noto Sans Telugu 2.000 ([f3de0d71c](https://github.com/googlefonts/noto-fonts/blob/f3de0d71c/phaseIII_only/unhinted/ttf/NotoSansTelugu/NotoSansTelugu-Regular.ttf))
